server/handlers/slack.js

```diff
@@ -95,7 +95,7 @@ module.exports = {
             data.user,
             {
               access_token: data.access_token,
-              scope: data.scope.split(','),
+              scope: data.scope ? data.scope.split(',') : '',
```

Comment:
This should be an empty list for consistency

Reply:
Good call. Updating now.